app/src/parser_impl.c

```diff
@@ -347,14 +348,19 @@ parser_error_t _checkVersions(parser_context_t *c) {
     transactionVersion += (uint32_t) p[2] << 16u;
     transactionVersion += (uint32_t) p[3] << 24u;
 
-    if (specVersion < SUPPORTED_MINIMUM_SPEC_VERSION) {
+    if (transactionVersion != (SUPPORTED_TX_VERSION_CURRENT) &&
+        transactionVersion != (SUPPORTED_TX_VERSION_PREVIOUS) ) {
         return parser_spec_not_supported;
     }
 
-    if (transactionVersion != (SUPPORTED_TX_VERSION)) {
-        return parser_tx_version_not_supported;
+    if (transactionVersion == (SUPPORTED_TX_VERSION_CURRENT) &&
+        specVersion < SUPPORTED_MINIMUM_SPEC_VERSION) {
```

Comment:
the minimal spec version typical was to avoid anyone using the app before the DOT redenomination

we should probably restrict the minimum version in both cases

Reply:
Done!